db/data_migration/20160211164516_republish_topical_event_about_pages_to_publishing_api.rb

```diff
@@ -0,0 +1 @@
+AboutPage.where.not(content_id: nil).each(&:publish_to_publishing_api)
```

Comment:
Well I can hardly fault you for following my example then! Still, what you've changed it to is better :+1: